src/app/plugins/platform/Docker/plugins/cms/Drupal8/index.ts

```diff
@@ -269,20 +273,84 @@ class Drupal8 extends Generator {
     cliEditor.addComposer('services/drupal');
   }
 
-  private async _installDrupal() {
+  /**
+   * Complete scaffolding and customization steps for the Drupal service directory.
+   */
+  private async _scaffoldDrupal(): Promise<void> {
     if (!this.shouldInstall) {
       return;
     }
 
-    this.debug('Triggering Drupal installation process.');
-    await installDrupal({
-      documentRoot: this.documentRoot,
-      projectType: this.projectType,
-      serviceDirectory: this.destinationPath('services'),
-    });
+    // Create the service directory if it doesn't exist.
+    // If the services directory doesn't exist, Docker fails since it can't mount
+    // it as a volume mount.
+    if (!this.existsDestination('services/drupal')) {
+      this.debug('Creating Drupal service directory.');
+      mkdirSync(this.destinationPath('services/drupal'), { recursive: true });
```

Comment:
Since you're already in an async function, avoid the `*Sync` fs functions: you can `util.promisify(fs.mkdir)` to get an `await`-able function.

Reply:
Thanks. Addressed in 44c8181.